local/bin/sh/helpers.sh

```diff
@@ -155,6 +155,9 @@ sync_integration_metrics() {
 	integrations_sync.py --token ${GITHUB_TOKEN}
 }
 
+update_pre_build() {
+    update_pre_build.py --token ${GITHUB_TOKEN}
+}
```

Comment:
let's decorate this with the `start_step`, `pass_step`, and `|| fail_step` functions. These will standardize the bash calls and report to datadog. see _most_ of the other helpers as examples.